ladybug/windrose.py

```diff
@@ -547,84 +585,55 @@ def frequency_lines(self):
 
         return [self._transform(Polygon2D.from_array(vecs)) for vecs in ytick_array]
 
-    @property
+    @ property
+    def windrose_lines(self):
+        """Get the windrose lines as Polygon2D lists."""
+
+        if self._poly_array is None:
+            _ = self.colored_mesh
```

Comment:
What is the reason for this?  Does calling self.colored_mesh make the _poly_array?
Maybe a comment stating this would help.

Reply:
Yes exactly. The colored_mesh method contains all the logic to check for various user-defined parameters when computing windrose geometry, so the geometric properties are set there. Added a comment.